source/_patterns/03-organisms/sections/block-group-2up.json

```diff
@@ -0,0 +1,41 @@
+{
+  "is_h2": true,
+  "block_media": {
+    "break_m": "500",
+    "img": {
+      "alt": "Alt Text",
+      "src_s": "https://picsum.photos/500/400?image=128",
+      "src_m": "https://picsum.photos/800/500?image=129",
+      "src_l": "https://picsum.photos/500/400?image=130"
+    }
+  },
+  "kicker": false,
+
+  "featured_block_1": {
+    "has_top_title": true,
+    "has_picture": true,
```

Comment:
Because when the blocks are in the 2up layout, the image size stretches a little too wide for performance on a smaller device, so I decided to progressively load a larger image after 500px or so. For non-2up, those blocks are all two columns (image left and content right) and they never go past 350px, so no need to load in a larger image.

Reply:
that makes sense for now, but I'd like these components to be compatible with sizes beyond the constrained widths they've been designed for here. can you swap in for a picture element, even if there aren't wider sizes to consider at this moment?

Reply:
Yeah, any block can use a picture element. They're just not needed for any of the other blocks since the images stay small. These have images that get pretty large, so I'm using the `<picture>` element here instead. It's a conditional in the block template. 

I guess we could always use the `<picture>` element and only have it contain one `img`? It's not necessarily what it's meant for, but probably wouldn't hurt anything.

Reply:
it'll be a little easier from my end b/c I only made a single `nypr-picture` atom and it's also the container for the logic I'll use to eventually implement lazy loading

Reply:
Ok, I'll refactor all blocks to use the `<picture>` tag even if it only contains one `<img>` element.